submissions/eve5ince/html-css-popup/css/master.css

```diff
@@ -0,0 +1,147 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	text-decoration: none;
+	list-style-type: none;
+	color: #757575;
+}
+body {
+	background: rgb(238,210,232);
+	background: linear-gradient(45deg, rgba(238,210,232,0.7889355571330094) 22%, rgba(99,252,69,0) 100%); 
+	font-family: Arial, Helvetica, sans-serif;
+}
+nav>ul {
+   display: flex;
+   justify-content: flex-end;
+	align-items: center;
+	font-size: 18px;
+	background-color: transparent;
+	padding: 5px 0 5px 0;
+	border-radius: 25px;
+	letter-spacing: 0.05em;
+}
+li {
+	margin: 30px 25px 0px 25px;
+}
+a {
+	padding: 2px;
+}
+.links-item:hover{
+	transition-duration: 0.22s;
+	opacity: 0.7;
+}
+.nav-img {
+	display: block;
+	background-repeat: no-repeat;
+	background-size: contain;
+	width: 32px;
+	height: 32px;
+}
+.control {
+	display: none;
+	position: absolute;
+	top: 100px;
+	right: 96px;
+	width: 320px;
+	height: 460px;
+	border: 2px groove;
+	background: transparent;
+	border: 1px solid;
+	overflow: auto;
+	-webkit-scrollbar: thin;
+	-webkit-scrollbar: #f9dcc4 transparent;
+}
+::-webkit-scrollbar {
+    width: 10px;
+}
+::-webkit-scrollbar-track {
+   -webkit-box-shadow: inset 0 0 4px rgba(0,0,0,0.3); 
+}
+
+::-webkit-scrollbar-thumb {
+    -webkit-box-shadow: inset 0 0 4px rgba(0,0,0,0.5); 
+}
+.links, .more {
+	display: grid;
+   grid-template-columns: repeat(3, 1fr);
+	font-size: 12px;	
+	text-align: center;
+	grid-gap: 10px;
+	padding: 0px 10px 0px 10px;
+}
+.links>a:focus, .more>a:focus {
+	 background: rgba(238,210,232, 0.2); 
+}
+img>p {
+	font-size: 14px;
+}
+label {
+	position: absolute;
+}
+input[type="checkbox"] { 
+	position: relative;
+	z-index: 0;      
+	appearance: none;
+	top: 2px;
+	left: 2px;
+	width: 28px;
+	height: 28px;
+}
+a:focus, input:focus {
+  outline: 1px solid #757575;
+}
+.bell {
+	width: 28px;
+	height: 28px;
+}
+#popUp {
```

Comment:
It is not necessary, you have already applied the value for `lable`

Reply:
Done